soh/soh/Enhancements/randomizer/randomizer_grotto.c

```diff
@@ -205,7 +208,8 @@ void Grotto_OverrideActorEntrance(Actor* thisx) {
 
         if (grottoContent == grottoLoadTable[index].content && gPlayState->sceneNum == grottoLoadTable[index].scene) {
             // Find the override for the matching index from the grotto Load List
-            // SaveFile_SetEntranceDiscovered(0x0700 + index);
+            Entrance_SetEntranceDiscovered(0x0700 + index);
+            EntranceTracker_SetLastEntranceOverride(0x0700 + index);
```

Comment:
similar to the `0x800` stuff, would be nice to not have a magic number for the grotto load list stuff